test/gtest/uct/ib/test_ib.cc

```diff
@@ -866,29 +867,38 @@ class test_uct_event_ib : public test_uct_ib {
         }
     }
 
-    void check_send_cq(uct_iface_t *iface, size_t val) {
-        uct_ib_iface_t *ib_iface = ucs_derived_of(iface, uct_ib_iface_t);
-        struct ibv_cq  *send_cq = ib_iface->cq[UCT_IB_DIR_TX];
-
-        if (val != send_cq->comp_events_completed) {
-            uint32_t completed_evt = send_cq->comp_events_completed;
-            /* need this call to acknowledge the completion to prevent iface dtor hung*/
-            ibv_ack_cq_events(ib_iface->cq[UCT_IB_DIR_TX], 1);
-            UCS_TEST_ABORT("send_cq->comp_events_completed have to be 1 but the value "
-                           << completed_evt);
+    size_t get_comp_events(const uct_iface_t *iface, const uct_ib_dir_t dir)
+    {
+        uct_ib_iface_t *ib_iface;
+#if IBV_HW_TM
```

Comment:
why `HW_TM`?

Reply:
It is just the only macro that fits my need that I've found. Do you have any suggestions for what it should be replaced?

Reply:
i'd suggest `HAVE_MLX5_DV`. `HW_TM` controls just tag offload capability